assets/js/modules/analytics/datastore/properties.js

```diff
@@ -407,6 +408,33 @@ const baseSelectors = {
 
 		return select( STORE_NAME ).isFetchingGetPropertiesProfiles( accountID );
 	} ),
+
+	/**
+	 * Gets all "classic" Universal Analytic and Google Analytics 4 (GA4) properties this account can access.
+	 *
+	 * Returns an array of all UA + GA4 analytics properties.
+	 *
+	 * Returns `undefined` if accounts have not yet loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state     Data store's state.
+	 * @param {string} accountID The Analytics Account ID to fetch properties for.
+	 * @return {(Array.<Object>|undefined)} An array of Analytics properties; `undefined` if not loaded.
+	 */
+	getPropertiesIncludingGA4: createRegistrySelector( ( select ) => ( state, accountID ) => {
+		const properties = select( STORE_NAME ).getProperties( accountID );
+
+		let propertiesGA4 = [];
+
+		if ( select( 'modules/analytics-4' ) ) {
+			propertiesGA4 = select( 'modules/analytics-4' ).getProperties( accountID );
+		}
+
+		const propertiesGA4Mapped = propertiesGA4.map( ( property ) => ( { ...property, id: property._id } ) );
```

Comment:
according to the fixtures (assuming these to be correct) `ua` properties have an `id` key, but `ga4` have an `_id` key